app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -41,6 +41,10 @@ def redirect_to_dashboard_if_submitted
       redirect_to candidate_interface_application_complete_path if current_application.submitted?
     end
 
+    def redirect_to_post_offer_dashboard_if_accepted
+      redirect_to candidate_interface_application_offer_dashboard_path if current_application.application_choices.map(&:status).include?('pending_conditions') && current_application.show_new_reference_flow?
```

Comment:
It's quite a long `if` condition for one line.
Maybe an `if` block or a method to return the boolean value of the condition to shorten things here?

Reply:
Done! ♻️